.github/workflows/debug-docs-test_coverage.yml

```diff
@@ -103,7 +103,7 @@ jobs:
     - name: test
       run: |
         cd ufs_utils/build/tests/chgres_cube/data
-        wget https://ftp.emc.ncep.noaa.gov/static_files/public/UFS/ufs_utils/unit_tests/chgres_cube/gfs.v16.sfc.history.nc
+        wget -i files.txt
```

Comment:
What does this do?

Reply:
I could not get 'wget' to work with wildcards. So, 'files.txt' is a list of files to download. If you know a better way, let me know.